Makefile

```diff
@@ -5,7 +5,11 @@ test-unit:
 	ansible-test units --docker -v --color --python $(PYTHON_VERSION) $(?TEST_ARGS)
 
 test-integration:
-	ansible-test integration --diff --no-temp-workdir --color --python $(PYTHON_VERSION) -v $(?TEST_ARGS)
+	ansible-test integration --docker --diff --color --python $(PYTHON_VERSION) -v $(?TEST_ARGS)
```

Comment:
not podman :))) ?

Reply:
--docker option falls back to podman - https://docs.ansible.com/ansible/latest/dev_guide/testing_running_locally.html#containers.  Maybe we can get the --docker option namechanged at some point :D